ncm-pbsclient/src/main/perl/pbsclient.pod

```diff
@@ -45,10 +41,11 @@ Two extra processor related flags can be specified : ncpus, and ncores
 ncpus is the number of physical CPUs in the node, and ncores is the total number of cores.
 ** All CPUs in one host are assumed to be the same **
 
-Example properties are : "ncores", "ncpus", "flags", "model name","cpu MHz","cpu family","model","stepping"
+Example properties are : "ncores", "ncpus", "flags", "model name", "cpu MHz",
+"cpu family","model","stepping"
```

Comment:
Any particular reason you've hard-wrapped these lines? They all seem to be less than 120 characters...

Reply:
not really, fell of my screen. I can revert if you want

Reply:
@jrha line split reverted